core/src/main/java/org/commonjava/maven/ext/core/impl/PluginRemovalManipulator.java

```diff
@@ -54,8 +54,10 @@
 
```

Comment:
I think we should make the session protected for reuse?

Reply:
Done.